### ISSUE TITLE
PeoplePicker suggestions not adopting high contrast colors

### ISSUE THREAD

**Issue by rebeccaballantyne** [OfficeDev/office-ui-fabric-react#4875]
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: High

Products/sites affected: All

#### Describe the issue:

1. Open a word document 
2. Navigate to "Share" button in the ribbon and press Enter to activate it
3. Observe that "Share" dialogue will open
4. Navigate to "People to share with" editable combo box and type content
5. Observe that the available suggestions are not High contrast theme colors(white & theme 2) - for example, text is not visible as it always remains black.

![available suggestions are not visible when focused by keyboard](https://user-images.githubusercontent.com/22757844/40076734-9504d434-5834-11e8-83a7-cef5d8965b21.png)

**Comment by BrendanMcK** [comment]
I'm seeing similar - and noting that the job title text is also not reflecting the HC selected text color. This is using a repro from the Fabric site:

![image](https://user-images.githubusercontent.com/1754373/40524505-a9bc8040-5f8f-11e8-82e5-25016b8fcc2c.png)

**Comment by dzearing** [comment]
@joschect This seems pretty important for accessibility specifically with the light HC mode; we could possibly pitch in here and help fix this, unless you have a fix incoming soon. It's been sitting since May 2018, and I suspect this impacts ODSP picker usage just like any other scenario.

**Comment by joschect** [comment]
@dzearing it looks like there is some logic that checks for the highcontrastslector in the peoplepickeritem styles, we could probably port some of that to to the suggestion and that could work. I don't have a fix coming soon though.

**Comment by Jahnp** [comment]
@khuynh92 would you mind taking a look at this when you have a chance?

**Comment by khuynh92** [comment]
Of course! I'll take a look at the issue as soon as possible.
